Header header

# Base mode
uint8 base_mode
# Custom mode
uint8 custom_mode
# Current mission
uint8 current_waypoint
uint8 system_status
#Total number of mission items
uint8 num_of_waypoints
